### ISSUE TITLE
Does mpv.net support the mpv property "focused"?

### ISSUE THREAD

**Issue by croak9854** [mpvnet-player/mpv.net#547]
**The Issue**
I tried to do the following using a lua script:
1. Pause playback automatically when alt+tab.
2. And resume playback when mpv.net regains focus.

To do this I used: mp.get_property_native("focused") and the script did not work. Then I used another script to OSD if window is focused and it always returned 'no' even when it was focused. Does mpv.net support this property and am I doing something wrong?

**To Reproduce**
To reproduce, try the following script:
```
local running = false
local timer
function show_focus_state()
    running = not running
    if running then
        timer = mp.add_periodic_timer(1, function()
            local focus = mp.get_property("focused")
            mp.osd_message(focus)
        end)
    else
        timer:stop()
    end
end
mp.add_key_binding("y", "show_focus_state", show_focus_state)
```
I used the same script in mpv and the osd was 'yes'. I also used the script I was using to pause the play on alt+tab on mpv and it worked there. 

**Additional context**
The script used to play only when focused:
```
was_pause = mp.get_property_native("pause")
function f_load(event)
	if was_pause == false then
		mp.set_property_native("pause", false)
	end
end
mp.register_event("file-loaded", f_load)
mp.observe_property("focused", "bool", function(name, value)
	local pause = mp.get_property_native("pause")
	local is_foc = mp.get_property_native("focused")
	if value == false then
		mp.set_property_native("pause", true)
		was_pause = false
	else
		if is_foc then
			mp.set_property_native("pause", false)
			was_pause = true
		else
			if was_pause then
				mp.set_property_native("pause", true)
			end
		end
	end
end)
```
1. mpv.net version: v6.0.3.0 
2. Windows version: 11

Am I doing something wrong? I am not experienced with any coding. Thank you.

**Comment by stax76** [comment]
No, but I want to try adding an option that will enable using the mpv video window instead of mpv.net's video window, with context menu support.

It might also be possible to add the capability to the mpv.net video window implementation.

And lastly, I might build a GUI extension for mpv, more info is here:

https://github.com/mpv-player/mpv/issues/5500#issuecomment-1680804458

https://github.com/mpv-player/mpv/issues/12185

**Comment by stax76** [comment]
I've made a feature request here:

https://github.com/mpv-player/mpv/issues/12676

**Comment by stax76** [comment]
I hope it's fixed in the beta release from today.
